### ISSUE TITLE
A small question about OCSVM

### ISSUE THREAD

**Issue by QiaoJohnny** [nomewang/M3DM#39]
@nomewang ,thank you very much for sharing your code! However, I have a small question about the code.
In the code, The definition of OCSVM says that
`self.detect_fuser = linear_model.SGDOneClassSVM(random_state=42, nu=args.ocsvm_nu,  max_iter=args.ocsvm_maxiter)`
`self.seg_fuser = linear_model.SGDOneClassSVM(random_state=42, nu=args.ocsvm_nu,  max_iter=args.ocsvm_maxiter)`
However, in the paper says that "For DLF, we use two linear OCSVMs with SGD optimizers, the learning rate is set as 1×10−4 and train 1000 steps for each class." 
So I wanna know how to set training rate in this code about OCSVM. Can you answer the question about it?
Thank you a lot again!
